CoffeeChat/Controller/DemographicsViewController.swift

```diff
@@ -0,0 +1,166 @@
+        view.addSubview(majorSearchBar)
+
+        hometownSearchBar.backgroundColor = .backgroundLightGray
+        hometownSearchBar.backgroundImage = UIImage()
+        // hometownSearchBar.delegate = self
+        let hometownSearchField = hometownSearchBar.value(forKey: "searchField") as? UITextField
+        hometownSearchField?.backgroundColor = .backgroundLightGray
+        hometownSearchField?.attributedPlaceholder = NSAttributedString(string: "Hometown", attributes: [NSAttributedString.Key.foregroundColor: UIColor.textDarkGray])
+        hometownSearchField?.clearButtonMode = .never
+        hometownSearchBar.setImage(UIImage(), for: .search, state: .normal)
+        hometownSearchBar.layer.cornerRadius = fieldsCornerRadius
+        hometownSearchBar.searchTextPositionAdjustment = UIOffset(horizontal: -6, vertical: 0)
+        hometownSearchBar.showsCancelButton = false
+        view.addSubview(hometownSearchBar)
+
+        pronounsTextField.backgroundColor = .backgroundLightGray
+        pronounsTextField.attributedPlaceholder = NSAttributedString(string: "Pronouns", attributes: [NSAttributedString.Key.foregroundColor: UIColor.textDarkGray])
+        pronounsTextField.leftView = UIView(frame: CGRect(x: 0, y: 0, width: 12, height: 49))
+        pronounsTextField.leftViewMode = .always
+        pronounsTextField.layer.cornerRadius = fieldsCornerRadius
+        view.addSubview(pronounsTextField)
+
+        nextButton.setTitle("Next", for: .normal)
+        nextButton.setTitleColor(.textBlack, for: .normal)
+        nextButton.backgroundColor = .backgroundLightGray
+        nextButton.layer.cornerRadius = 26
+        nextButton.addTarget(self, action: #selector(nextButtonPressed), for: .touchUpInside)
+        view.addSubview(nextButton)
+
+        setUpConstraints()
+    }
+
+    @objc func nextButtonPressed() {
+        delegate.nextPage(index: 1)
+    }
+
+    func setUpConstraints() {
+
+        let helloLabelHeight: CGFloat = 30
+        let helloLabelSpacing: CGFloat = 100
+        let nextBottomPadding: CGFloat = 90
+        let nextButtonSize = CGSize(width: 225, height: 54)
+        let textFieldSidePadding: CGFloat = 40
+        let textFieldHeight: CGFloat = 49
+        let textFieldTopPadding: CGFloat = 20
+
+        helloLabel.snp.makeConstraints { make in
+            make.centerX.equalToSuperview()
+            make.height.equalTo(helloLabelHeight)
+            make.top.equalTo(view.safeAreaLayoutGuide).offset(helloLabelSpacing)
+        }
+
+        greetingLabel.snp.makeConstraints { make in
+            make.centerX.equalToSuperview()
+            make.top.equalTo(helloLabel.snp.bottom).offset(10)
+        }
+
+        classTextField.snp.makeConstraints{ make in
+            make.centerX.equalToSuperview()
+            make.top.equalTo(greetingLabel.snp.bottom).offset(149)
+            make.left.equalToSuperview().offset(textFieldSidePadding)
+            make.right.equalToSuperview().offset(-textFieldSidePadding)
```

Comment:
remember to use `leading` and `trailing` instead of `left//right`. you can also condense this into:
`make.leading.trailing.equalToSuperview().inset(textFieldSidePadding)`